helm/cluster-apps-operator/templates/rbac.yaml

```diff
@@ -51,6 +51,14 @@ rules:
       - delete
       - get
       - list
+  - apiGroups:
+      - ""
+    resources:
+      - events
```

Comment:
Do we need `events` permission for this change?

Reply:
I'm pretty sure there were errors in the logs. Are the events coming from operatorkit? https://github.com/giantswarm/operatorkit/blob/8b4f7dbf932c3152fd0f990cc73bf6fc3ecee79a/docs/using_kubernetes_events.md

I'll check in my next cluster creation test and if no events are created I'll remove this.